django_fsm/__init__.py

```diff
@@ -426,7 +426,6 @@ def is_field_transition_method(attr):
                     attr._django_fsm.field in [self, self.name]
                     or (
                         isinstance(attr._django_fsm.field, Field)
-                        and issubclass(self.model, attr._django_fsm.field.model)
```

Comment:
And there was otherwise no failing test for this test to exist ?!?! Do you know @kmmbvnr what this was aimed to handle?

Reply:
Well, if it still works for the rest of @pfouque setup, why not try it. In the worst case, it will brake somewhere else, we will have a failing test then and fix it again :-)